book/04-behind-atom/sections/07-maintaining-your-packages.asc

```diff
@@ -0,0 +1,33 @@
+=== Maintaining Your Packages
+
+While publishing is, by far, the most common action you will perform when working with the packages you provide, there are other things you may need to do.
+
+==== Unpublish a Version
+
+If you mistakenly published a version of your package or perhaps you find a glaring bug or security hole, you may want to unpublish just that version of your package. For example, if your package is named `package-name` and the bad version of your package is v1.2.3 then the command you would execute is:
+
+```bash
+apm unpublish package-name@1.2.3
+```
+
+This will remove just this particular version from the https://atom.io package registry. Anyone who has already downloaded this version will still have it, but it will no longer be available for installation by others.
+
+==== Adding a Collaborator
+
+Some packages get too big for one person. Sometimes priorities change and someone else wants to help out. You can let others help or create co-owners by https://help.github.com/articles/adding-collaborators-to-a-personal-repository/[adding them as a collaborator] on the GitHub repository for your package. *Note:* Anyone that has push access to your repository will have the ability to publish new versions of the package that belongs to that repository.
+
+You can also have packages that are owned by a https://help.github.com/articles/creating-a-new-organization-account/[GitHub organization]. Anyone who is a member of the organization that has push access to the repository for the organization's package will be able to publish new versions of the package.
```

Comment:
> Anyone who is a member of the organization that has push access to the repository for the organization's package will be able to publish new versions of the package.

Since permissions are managed via teams, here's a slightly different version which mentions teams:

> Anyone who is a member of an organization's https://help.github.com/articles/permission-levels-for-an-organization-repository/[team] which has push access to the package's repository will be able to publish new versions of the package.